src/components/ui/RenderIf.tsx

```diff
@@ -6,3 +6,9 @@ interface RenderIfProps {
 
 export const RenderIf = ({ condition = false, children }: PropsWithChildren<RenderIfProps>) =>
   condition ? <>{children}</> : null;
+
+export const RenderIfElse = (props: {
```

Comment:
nit: Honestly, I do not see much benefit of those components. But I am even ok with keeping them.

Reply:
I would just not use it for more complex subtrees because of performance reasons (both branches are evaluated every time).

Reply:
No strong opinion here.